Automattic/wp-calypso
# Domains: Add plan upsell in domain settings and nameserver settings for non primary domain

#### Changes proposed in this Pull Request

This PR adds a plan upsell card in the domain settings and also in the Name Servers and DNS view for a non primary domain, as part of this test: p99Zz8-10j-p2

<img width="730" alt="Screenshot 2020-02-27 at 2 29 06 PM" src="https://user-images.githubusercontent.com/13062352/75449601-8b347680-596d-11ea-842e-9778cd0f1f2d.png">

#### Testing instructions

For a site which has at least one registered domain:

* Make sure the upsell only shows for registered domains
* Verify it won't show for primary domains, and non primary mapped or WPCOM domains
* Verify it won't show when the domain is not pointing to our servers (either by hacking the backend or by adding an A record)
* Check that the same upsell also shows when clicking "Name Servers and DNS"
* Verify nothing gets broken :)
* Verify that it works more or less the same with and without this flag `domains/new-status-design`
